A tutorial using simulated Gaussian data

XueqingYin opened the pull request:
* **Summary of changes** (Bug fix, feature, docs update, ...)

This is to create a tutorial that covers simulating Gaussian data, fitting the BHM and exploring the key functionality in fdmr.


* **Please check if the PR fulfills these requirements**

- [ ] Closes #131 #112 
- [ ] Tests added and passed
- [ ] All code checks passing - `styler` run over code
- [x] Vignettes updated/added
- [ ] Added an entry in the latest `CHANGELOG.md` file if fixing a bug or adding a new feature
- [ ] Any new libraries added to `DESCRIPTION`

mnky9800n commented on vignettes/Simulation_GaussianDat.Rmd:
````diff
@@ -0,0 +1,275 @@
+```{r hist,error=TRUE,fig.cap="A histogram of the simulated Gaussian data.", fig.width=8, fig.height=4, fig.align = "center"}
+hist(simdf$datn,main='', xlab='Data')
+```
+
````
i think here you should put a map of the data too

mnky9800n reviewed (changes requested):
i think you should have a map of the raw data before you start modeling the data.

mnky9800n reviewed (approved):
love the map

mnky9800n commented:
```
Caused by error:
! Invalid dataset, the folder /home/runner/fdmr/tutorial_data/priors does not exist.
Backtrace:
    ▆
 1. └─fdmr::load_tutorial_data(dataset = "priors", filename = "spatial_dataBris.rds")
 2.   └─fdmr::get_tutorial_datapath(dataset = dataset, filename = filename)
Execution halted
Error: Process completed with exit code 1.
```

XueqingYin commented:
@mnky9800n That's strange. I have no issues running ```sp_data <- fdmr::load_tutorial_data(dataset = "priors", filename = "spatial_dataBris.rds")```.  "spatial_dataBris.rds" is saved in the fdmr_data repo in "priortut_data.tar.bz2 (https://github.com/4DModeller/fdmr_data/blob/main/priortut_data.tar.bz2). 

Xueqing

XueqingYin commented:
@mnky9800n  I know why it happens. I should have retrieved the priors data first before loading in the data. It's now fixed. I added a line of code that uses the ```fdmr::retrieve_tutorial_data``` function to retrieve the data first from fdmr data store before loading. Could you please try again and let me know if it works?

Xueqing
